### DISCUSSION TITLE
the direction of the arrows is always left to right

### DISCUSSION THREAD

**Discussion by lexoring** [antvis/X6#3118] [Q&A] [answered]
Hi! You have a great script! 
How to make it so that no matter which side the edge starts from, the arrow is always from left to right?

I made a video, this is an example from another similar script, here is a link how it was done. On another script https://reactflow.dev/docs/examples/nodes/dynamic-grouping/, there is a separation between the source and the target, and there the arrow always goes to the target. And how to do it here, tell me, please. Thank you!

**I would like this behavior of the arrow:**

[recording (2).webm](https://user-images.githubusercontent.com/74623056/211172442-13665adb-780c-47a0-9f46-0ec077dc26f5.webm)

_____________________________________

**and here is how it is done in ant:**

[recording (3).webm](https://user-images.githubusercontent.com/74623056/211172468-ec81227c-e8f5-40dc-af0f-ebcb4b6c612e.webm)

**Comment by NewByVector** [comment, marked as answer]
In fact, this feature is more business semantics, not general purpose, because I have no way to know which node is target node, only the developer knows. x6 can do this by modifying the arrows of the wires in the `edge:connected` event, as long as it knows which node is target node.

**Reply by lexoring** [reply]
thank you, I did so
